packages/unmock-core/src/__tests__/hash/v0/index.test.ts

```diff
@@ -1,187 +1,204 @@
-import { defaultOptions, ignoreAuth } from "unmock-core";
+import { ignoreAuth } from "unmock-core";
 import computeHash, { makeHashable } from "../../../hash/v0";
 
 describe("Hash function", () => {
```

Comment:
I would recommend checking this file out and then working on it again after the `hash` merge, as the `hash` tests are all new in the `hash` branch.